pkg/volume/csi/csi_attacher.go

```diff
@@ -282,8 +282,7 @@ func (c *csiAttacher) MountDevice(spec *volume.Spec, devicePath string, deviceMo
 	klog.V(4).Infof(log("attacher.MountDevice(%s, %s)", devicePath, deviceMountPath))
 
 	if deviceMountPath == "" {
-		err = fmt.Errorf("attacher.MountDevice failed, deviceMountPath is empty")
-		return err
+		return errors.New("attacher.MountDevice failed, deviceMountPath is empty")
```

Comment:
Can you also wrap all the messages with `log()` to be consistent with the rest?

Reply:
@msau42 apologies for the delay. done